api-reference/upload-file-api/server-side-file-upload.md

```diff
@@ -310,14 +315,23 @@ $imageKit = new ImageKit(
     $url_end_point
 );
 
-$base64Img = "iVBORw0KGgoAAAAN";
+$img = file_get_contents(__DIR__."/image.jpg");
+
+// Encode the image string data into base64
+$base64Img = base64_encode($img);
+
+// Options
+$options = [
+    "tags" => implode(",", ["tag1", "tag2"]),
+    "customCoordinates" => implode(",", ["10", "10", "100", "100"])
+];
 
 // Upload Image - base64
-$uploadFile = $imageKit->upload(array(
-    'file' => $base64Img,
-    'fileName' => "my_file_name.jpg",
-    "tags" => implode(",", array("t-shirt", "summer", "men"))
-));
+$uploadFile = $imageKit->upload([
+    "file" => $base64Img,
+    "fileName" => "my_file_name.jpg",
+    "options" => $options
```

Comment:
Why are we not mering all options in the same array in parallel to `file` and `fileName` parameter. The old specification looks more simple. I think we should change this in SDK and here in docs.